data/auto-apply-sequence-report.md

```
@@ -0,0 +1,390 @@
+# Secuencia de Auto-Apply
+
+Fecha: 2026-04-15T17:20:31.794Z
+
+## test-auto-apply
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)